tests/auth_check.rs

```diff
@@ -1,39 +1,32 @@
 //! Rust integration test to verify that invoking the restricted create CA
 //! REST API requires a valid bearer token.
-use std::str::FromStr;
 
-use rpki::ca::idexchange::Handle;
+use hyper::StatusCode;
+use krill::commons::util::httpclient;
 
-use krill::{
-    commons::api::Token,
-    test::{
-        init_ca, mem_storage, start_krill_with_custom_config, test_config,
-    },
-};
+mod common;
 
-extern crate krill;
 
 #[tokio::test]
-#[should_panic]
 async fn auth_check() {
-    // Use a copy of the default test Krill config but change the server admin
-    // token thereby hopefully causing the bearer token sent by the test
-    // suite support functions not to match and thus be rejected which in turn
-    // should cause a Rust panic.
-    let storage_uri = mem_storage();
-    let mut config =
-        test_config(&storage_uri, None, false, false, false, false);
-    config.admin_token = Token::from("wrong secret");
+    let (server, _tempdir) = common::KrillServer::start().await;
 
-    // Start Krill with the customized config
-    start_krill_with_custom_config(config).await;
+    // Get a client and change its auth token.
+    let mut client = server.client().clone();
+    client.set_token("different token".into());
```

Comment:
"different" doesn't imply "wrong" to me, perhaps replace "different token" with "deliberately incorrect token"?

Reply:
I’ll change it to “wrong secret”.